src/components/layout/ContactsSelectionBar.jsx

```diff
@@ -22,13 +23,28 @@ class ContactsSelectionBar extends Component {
         actions={{
           trash: {
             action: () => {
+              const contactsWithAccounts = selection.filter(
+                contact => getConnectedAccounts(contact).length > 0
+              )
+              const allContactsConnected =
+                contactsWithAccounts.length === selection.length
+              const someContactsConnected =
```

Comment:
nit: I would have used `selection.every` and `selection.some`

Reply:
Good point, I rarely think of those.